src/neuro_relay/server.py

```
@@ -189,56 +222,81 @@ async def _handle_intermediary_forward(self, msg: dict):
+                            print(
+                                f"[Nakurity Backend] sent action to {integration_name}",
+                            )
+
+                            recev = await self._recv_q.get()  # must be filled by the receiver. here its filled in run_server()
 
-                            print("[Nakurity Backend] ... RESPONSE FROM INTEGRATION: ", recev)
+                            print(
+                                "[Nakurity Backend] ... RESPONSE FROM INTEGRATION: ",
+                                recev,
+                            )
                             return recev
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)